jest.conf/visual.load-test-components.js

```diff
@@ -48,3 +49,4 @@ Vue.component('CardsVisualTest', CardsVisualTest);
 Vue.component('KBreadcrumbs', KBreadcrumbs);
```

Comment:
Is this `KBreadcrumbs` import still needed now that we have the `KBreadcrumbsVisualTest` import, or can it be removed?

Reply:
Oh, it was never really necessary, removed! Thanks @LianaHarris360!